src/Symfony/Component/Validator/Constraints/Locale.php

```diff
@@ -29,4 +29,13 @@ class Locale extends Constraint
 
     public $message = 'This value is not a valid locale.';
     public $canonicalize = false;
+
+    public function __construct($options = null)
+    {
+        if (!($options['canonicalize'] ?? false)) {
+            @trigger_error('The "canonicalize" option with value `false` is deprecated since Symfony 4.1. Set it to `true` instead.', E_USER_DEPRECATED);
```

Comment:
Instead of backticks, you should use `"`, and make one sentence;

`The "canonicalize" option with value "false" is deprecated since Symfony 4.1, set it to "true" instead.`

Reply:
Message updated. Thank you.